cas-server-support-spnego/src/main/java/org/jasig/cas/support/spnego/authentication/handler/support/JcifsConfig.java

```diff
@@ -107,24 +112,30 @@ public void afterPropertiesSet() throws Exception {
         logger.debug("configured login configuration path : {}", propValue);
     }
 
+
     /**
      * Sets the jcifs service password.
      *
      * @param jcifsServicePassword the new jcifs service password
      */
     public void setJcifsServicePassword(final String jcifsServicePassword) {
-        logger.debug("jcifsServicePassword is set to *****");
-        Config.setProperty(JCIFS_PROP_SERVICE_PASSWORD, jcifsServicePassword);
+        if (StringUtils.isNotBlank(jcifsServicePassword)) {
```

Comment:
Why testing if the parameter is not blank in the setter? It's the first time we do that in setters, don't we? Is there any particular reason?